### ISSUE TITLE
Inconsist slate AST when transforming empty strings

### ISSUE THREAD

**Issue by rodikh** [inokawa/remark-slate-transformer#129]
I've recently updated remark-slate-transformer from v0.2.1 to v0.7.3 and that introduced a breaking change for me with the generated Slate AST.

Previously, parsing an empty string (or a string with only whitespaces) would generate an empty array as a result of processSync.

```js
const processor = unified().use(markdown).use(remarkToSlate);
const value1 = processor.processSync('').result; // => []
const value2 = processor.processSync(' ').result; // => []
const value3 = processor.processSync('val').result; // => [{"type": "paragraph", "children": [{"text": "val"}]}]
```

After updating remark-slate-transformer I am receiving a slightly different response.
```js
const processor = unified().use(markdown).use(remarkToSlate);
const value1 = processor.processSync('').result; // => [[{"text": ""}]]
const value2 = processor.processSync(' ').result; // => [[{"text": ""}]]
const value3 = processor.processSync('val').result; // => [{"type": "paragraph", "children": [{"text": "val"}]}]
```
These two results are not marked as `paragraph` nodes, which makes Slate freak out.

Is this the expected behavior? how do I make sure that remark-slate-transformer returns a paragraph node as root if nothing else is parsed?

**Comment by inokawa** [comment]
It is the behavior introduced in #42 .
The bug was caused by empty list, so it might be better to fix only list not all nodes.

**Comment by inokawa** [comment]
Published in `0.7.4`.

**Comment by rodikh** [comment]
Thanks for the quick fix @inokawa.
I can confirm that it works properly and slate is no longer confused :)
